reconcile/change_owners/diff.py

```diff
@@ -251,8 +251,6 @@ def build_jsonpath_part(element: str | int) -> jsonpath_ng.JSONPath:
             case int():
                 return jsonpath_ng.Index(element)
             case str():
-                if "." in element:
-                    return jsonpath_ng.Fields(f"'{element}'")
```

Comment:
is there a test case to cover that element with `.` now auto quoted in new version? https://github.com/h2non/jsonpath-ng/releases/tag/v1.6.0